pkg/agent/agent.go

```diff
@@ -231,14 +226,18 @@ func (a *Agent) bootstrap() error {
 		a.baseSVIDKey = key
 
 		// If we're here, we need to attest/Re-attest
-		regEntryMap, err := a.attest()
-		if err != nil {
-			return err
-		}
-		err = a.FetchSVID(regEntryMap, a.BaseSVID, a.baseSVIDKey)
+		regEntries, err := a.attest()
 		if err != nil {
 			return err
 		}
+		uri := a.config.TrustDomain
+		uri.Path = "spiffe/cp"
```

Comment:
This change will be visible to everything else that uses the same `Config` that was passed into `New()`.  Unexpected behavior at the very least, IMO.